### PR TITLE
Update graspy

### PR DESCRIPTION
**Pull request by loftusa**
1. add redirects for MRI
2. add mike, remove bear/cencheng
3. neuroparc to website
4. graspy pubs update

### CODE REVIEW

File: misc/redirect.py [hunk 1]
```diff
@@ -34,15 +34,16 @@
...
-    ['project/projectomes', 'mri-cloud/'],
+    ['project/projectomes', 'mri/'],
     ['data', 'ocp/'],
     ['RerF', 'sporf/'],
     ['randomerforest', 'sporf/'],
     ['ndreg', 'reg/'],
     ['workshop', 'about/workshops/'],
     ['ndcloud', 'ocp/'],
-    ['about/conduct', 'about/agreements/#respect'],
+    ['about/conduct', 'about/agreements/#respect/'],
```
Comment (falkben) [inline misc/redirect.py]: no trailing slash here

File: content/projects/connectomes.html [hunk 2]
```diff
@@ -18,14 +18,22 @@ <h2>Connectomes</h2>
 
 <div class="doc-nav">
     <ul>
-        <li><a href="#about"><h5>About</h5></a></li>
+        <li>
```
Comment (falkben) [inline content/projects/connectomes.html]: anything actually change in this file?

File: content/pubs/pubs.bib [hunk 3]
```diff
@@ -108,7 +108,7 @@ @article{bagging2019
 @article{graspy2019,
   title={GraSPy: Graph Statistics in Python},
   author={Chung, Jaewon and Pedigo, Benjamin D. and Bridgeford, Eric W. and Varjavand, Bijan K. and Vogelstein, Joshua T.},
-  journal={arXiv},
+  journal={Journal of Machine Learning Research},
   year={2019},
   month={april},
```
Comment (falkben) [inline content/pubs/pubs.bib:113]: not the right month. maybe it's fine. @j1c @bdpedigo would know?

**Review by falkben — CHANGES_REQUESTED** [review]
see comments

File: content/pubs/pubs.bib (hunk 3 above)
Comment (j1c) [reply content/pubs/pubs.bib:113]: Its not yet actually published tho

File: content/projects/connectomes.html (hunk 2 above)
Comment (loftusa) [reply content/projects/connectomes.html]: Yes - `<a href="{{g.url('/content/projects/mri.html').path}}">Projectomes</a>.</p>` was `/projects/mri-cloud.html` before. Just turned off autoformatting as well, and reupdated file to have old formatting

File: content/pubs/pubs.bib (hunk 3 above)
Comment (loftusa) [reply content/pubs/pubs.bib:113]: @j1c jovo wanted me to update it anyway, since it's set to be published

**Comment by loftusa** [comment]
reverted the formatting from the autoformatter to make the `diff` easier to read as well

File: content/projects/mri.html [hunk 4]
```diff
@@ -15,12 +15,14 @@
...
+  link: https://neurodata.io/ndmg
+- name: NeuroParc
+  link: https://github.com/neurodata/neuroparc
 - name: Science in the Cloud (sic)
   link: https://github.com/neurodata/sic
-- name: NeuroData's MRI Graphs pipeline (ndmg)
-  link: https://github.com/neurodata/ndmg
 - name: NeuroData's GRaph with rich attribUTE DataBase (ndgrutedb)
-  link: https://github.com/neurodata/ndgrutedb
+  link: https://github.com/neurodata/ndgrutedbb
```
Comment (falkben) [inline content/projects/mri.html]: you have an extra b

**Review by falkben — CHANGES_REQUESTED** [review]
typo

File: content/projects/mri.html (hunk 4 above)
Comment (loftusa) [reply content/projects/mri.html]: thanks, good catch

**Comment by loftusa** [comment]
@falkben alright, should be ready to merge

**Review by falkben — APPROVED** [review]

**Pull request merged**